setup.py

```diff
@@ -29,16 +29,16 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'click',
-        'docker-py',
+        'click==6.6',
```

Comment:
It's fine to pin the requirements in requirements.txt, but not needed here unless there's some serious reason

Reply:
Yep, there is a reason, today `pip install shub` totally ignores `requirements.txt` because `setup.py` doesn't import the requirements from `requirements.txt`. So even though we pin libs there it doesn't have any effect.

We don't notice it during CI because `tox.ini` is explicitly referring to the `requirements.txt`.

Also, we need `tqdm==4.11.2` because newer versions generates a little bit different progress bars and we have test cases strictly defined for this progress bars, tests are failing with newer `tqdm`.

I thinking about importing `requirements.txt` to `setup.py` and we centralize the lib control on `requirements.txt`, WDYT? :)

Reply:
@vshlapakov I have added a commit[1] that loads from `requirements.txt`, so now `pip install shub` is going to load from it. Let me know if I'm doing something in the wrong way here.. thx

[1] - https://github.com/scrapinghub/shub/pull/383/commits/af6236916c5575f54ac7f0dfe836382256b25ae8